base-requirements.txt

```diff
@@ -1,46 +1,47 @@
 dj-database-url==0.5.0
-django-pipeline==1.6.14
-django-sitetree==1.10.0
-Django==2.0.13
+django-pipeline==2.0.6
+django-sitetree@https://github.com/idlesign/django-sitetree/archive/a2d77e9c282fc4be60698a8421c42a494393e794.zip
```

Comment:
Same thing here. Can't we use a more specific tag?

Reply:
This points to the last commit on the master as there were some fixes not yet shipped in a regular release, however, 4 days ago the v1.17.0 release was made, so I will move to the correct version number.

Reply:
fixed